cli/build.rs

```diff
@@ -11,18 +192,6 @@ fn main() {
     deno_typescript::ts_version()
   );
 
-  let c = PathBuf::from(env::var_os("CARGO_MANIFEST_DIR").unwrap());
-  let o = PathBuf::from(env::var_os("OUT_DIR").unwrap());
-
-  let root_names = vec![c.join("js/main.ts")];
-  let bundle = o.join("CLI_SNAPSHOT.js");
-  let state = deno_typescript::compile_bundle(&bundle, root_names).unwrap();
-  assert!(bundle.exists());
-  deno_typescript::mksnapshot_bundle(&bundle, state).unwrap();
-
-  let root_names = vec![c.join("js/compiler.ts")];
-  let bundle = o.join("COMPILER_SNAPSHOT.js");
-  let state = deno_typescript::compile_bundle(&bundle, root_names).unwrap();
-  assert!(bundle.exists());
-  deno_typescript::mksnapshot_bundle_ts(&bundle, state).unwrap();
+  create_main_snapshot();
+  create_compiler_snapshot();
```

Comment:
The problem with having a bunch of code in build.rs is that none of it can be unit tested. This code will only be tested through the build process. It seems we should strive to keep it as small as possible.